fossunited/chapters/doctype/foss_chapter/foss_chapter.json

```diff
@@ -11,6 +11,7 @@
   "is_published",
   "column_break_pmbj",
   "route",
+  "represent_image",
```

Comment:
What is representing image

Reply:
I have added a dummy image for every foss chapter city to represent it. As Ajzal suggested. For now it's just a default dummy image, But they're working on scapes for different cities. 

So represent_image = image representing a city.